rpc/rpc-lib/src/protocol-utils.h

```diff
@@ -6,11 +6,11 @@
 static inline int
 get_vol_type(int type, int dist_count, int brick_count)
 {
-    if ((type != GF_CLUSTER_TYPE_TIER) && (type > 0) &&
-        (dist_count < brick_count))
-        type = type + GF_CLUSTER_TYPE_MAX - 1;
+    if ((type == GF_CLUSTER_TYPE_TIER) || (type <= 0) || (dist_count <= 1) ||
+        (dist_count >= brick_count))
```

Comment:
I don't see when this could happen. Is this a safety check or does it cover some real case ?

Reply:
I believe the condition checks are just safety checks, the values are returned by glusterd to the cli. If somehow correct values are not populated then these checks would be helpful.